### PR TITLE
Kinetic Accelerator changes

### PR DESCRIPTION
**Pull request by Djiq**
## About The Pull Request

Kinetic accelerator now deals 40 damage to **only** /mob/living/simple_animal/hostile 

Otherwise it always deals 10 damage.

Unless you modify it with the low pressure mod, in which case it deals 20 damage.

## Why It's Good For The Game

No more bullshit with kinetic accelerator being the best weapon on the station as long as you are in low pressure.

## Changelog
:cl:
balance: KA now deals 40 damage to monsters, and 10 damage to everything else. Low pressure mod makes it deal 20 damage.
/:cl:

### CODE REVIEW

**Comment by Ghilker** [comment]
Just make it as it was before

**Comment by EdgeLordExe** [comment]
> Just make it as it was before

no

**Comment by Ghilker** [comment]
Why not?

**Comment by EdgeLordExe** [comment]
> Why not?

make a fix pr otherwise, this is completely unrelated to KA's being broken due to the projectile refactor.

**Comment by Fikou** [comment]
this is even more snowflake then pressure stuff, and youre making 10 tc make an extremely shitty laser gun that can fit like 1 more upgrade
ive never seen anyone do something like drain the air from the station to do funny murderbone

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm [hunk 1]
```diff
@@ -172,31 +172,34 @@
...
-			name = "weakened [name]"
-			damage = damage * pressure_decrease
-			pressure_decrease_active = TRUE
+
+	if(kinetic_gun)
+		var/list/mods = kinetic_gun.modkits
+		for(var/obj/item/borg/upgrade/modkit/M in mods)
+			M.projectile_prehit(src, target, kinetic_gun)
+
+	if(istype(target,/mob/living/simple_animal/hostile))
```
Comment (Qustinnus) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm:197]: look at the top of his head

**Comment by EdgeLordExe** [comment]
> this is even more snowflake then pressure stuff, and youre making 10 tc make an extremely shitty laser gun that can fit like 1 more upgrade
> it also makes pka's extremely good against slaughter demons, any syndie npc's, eldritch demons, just enabling the validhunter miner
> ive never seen anyone do something like drain the air from the station to do funny murderbone

good note, should be limited to only megafauna and asteroid monsters. not all /hostile. im gonna change that in a moment.

**Comment by Fikou** [comment]
> good note, should be limited to only megafauna and asteroid monsters. not all /hostile. im gonna change that in a moment.

bringe

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/projectiles/guns/energy/kinetic_accelerator.dm:197]: look at his lips 😳

**Comment by Sparkezel** [comment]
Honestly I think that the bug that happened to PKAs is actually quite a good thing, teaches you to get a PKA from aux base first and to take on miners using better ranged weaponry because pka still requires you to get into pretty much melee range.

**Comment by Fakka6969** [comment]
"Otherwise it always deals 10 damage."
"Unless you modify it with the low pressure mod, in which case it deals 20 damage."

Wdym always 10 dmg? Even with addons?

I tide 3x dmg PKA (before bug) not to fight (because it sucks, even with addons, and even in low pressure it will consistently lose to meta-gear) but because it breaks lockers, r-windows and airlocks. its the kind of 1min-emergency opener for alot of things and thats p much the only thing I really used it for apart from hitting ppl for fun.

Its like in between being a weapon and a tool, it has attributes of both but does no job good at all, especially as weapon its inferior to nearly anything people antag or validhunt with and doesnt deserve a nerf.

afaik theres a bug rn so that you can two-shot ppl with it even in non-low-pressure environment or go like shoot->baton to down even sec semi-instantly. just fix that bug. the pka will go back into obscurity where it came from.

I dont think experienced miners even use the fucking thing, the only thing where I saw it consistently coming to mind is blobs at which point... why not just do a hiero speedrun in something like 5 minutes and come back with a weapon 10x more broken against blob, even in low-pressure, than PKA?

So, to reiterate, its my worthless (i dont c*de) opinion that PKA sucks, especially if you view it singularly as weapon, and especially in the usual circumstances you will fight in. if you are planning to nerf it to 10 dmg 20 dmg even with syndie addon flat, absolutely noone will ever pick this thing up for anything ever again because it will be weaker than a sharpened kitchen cleaver from chef apart from newbs who dont know better and have no idea that its a still shitty unicorn to farm simplemobs now. its not used by most experienced players as it is, very few people appreciate its flexibility at all because theres better for mining, better for fighting, better for fighting mobs and better for tools. everything the PKA does, other stuff does tons better, for every single job imaginable, the only "gain" with PKA is that you have to carry less, but thats not worth losing fights, needing minutes to open door or shoot r-glass or being weaker against megafauna. Even dualwield antag PKA people with max upgrades in low pressure can easily lose versus a single decently geared and non-retarded seccie. and thats prob the only scenario in which it really seems like a decent choice for weapon at all apart from mobs. the only reason i tide pka is because im lazy enough to choose it over getting proper gear for anything sometimes, also fuck mining.

so yeah.. who killed u that u wanna nerf it this hard

where did uncle touch you with the pka

**Comment by Tlaltecuhtli** [comment]
#55495 fixes this issue without adding cocaine induced balancing

**Comment by Jared-Fogle** [comment]
how often is pka being good at low pressure used on the station, really? seems like you're just gutting a piece of environmental combat, which i think we should favor

**Review by Mothblocks — CHANGES_REQUESTED** [review]
👎 environmental combat is the best part of ss13 for me, blocking this unless another maintainer feels strongly about it

**Comment by necromanceranne** [comment]
It's basically only used for space fighting or when you've blasted a room to pieces and someone steps into a space tile, opening them up to the PKA.

You can definitely set up areas for us with a PKA but you need explosives first. Using siphons is usually too slow. 

It's also used against blobs simply because they habitually drain atmos from a room due to smashing down hulls.

**Comment by Timberpoes** [comment]
I second what Jared is saying. I don't see the PKAs low atmosphere strength as inherently bad.

Unlike Jared I'm just going to close this because I don't see it ever getting merged and we have too many open PRs heading into the holiday period right now.

**Pull request closed without merging**